client-and-server/server/php/composer.lock

```diff
@@ -1,75 +0,0 @@
-{
-    "_readme": [
-        "This file locks the dependencies of your project to a known state",
-        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
-        "This file is @generated automatically"
-    ],
-    "content-hash": "66b83a46a4be69d4204be618354c6181",
-    "packages": [
-        {
-            "name": "stripe/stripe-php",
-            "version": "v7.32.0",
-            "source": {
-                "type": "git",
-                "url": "https://github.com/stripe/stripe-php.git",
-                "reference": "edffb6d9370ea84f01faab0f6874b0e049e9df05"
-            },
```

Comment:
Why the change to the composer.lock for the php server?

Reply:
Good question. I don't think this should've been deleted!

Reply:
Looks like github is confused. This was moved from `client-and-server/server/php` to `server/php`